web/client/api/CSW.js

```diff
@@ -433,7 +461,7 @@ const Api = {
     workspaceSearch: function(url, startPosition, maxRecords, text, workspace) {
         return new Promise((resolve) => {
             require.ensure(['../utils/ogc/CSW', '../utils/ogc/Filter'], () => {
-                const {Filter} = require('../utils/ogc/Filter');
+                const { Filter } = require('../utils/ogc/Filter');
```

Comment:
why have you started to add all of this formatting? 
is there a tool you are using or what? 

let's avoid introducing custom syntax formation different from what setup with vscode or the eslint rules of this project, so please let me know if you can disable this